Building error

On Ubuntu 16.04 the make stops during Model.cpp-o process 
```
~/glabels-qt/build$ make
Scanning dependencies of target glbarcode
[  0%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/Factory.cpp.o
[  0%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/Barcode.cpp.o
[  1%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/Barcode1dBase.cpp.o
[  1%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/Barcode2dBase.cpp.o
[  1%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodeCode39.cpp.o
[  1%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodeCode39Ext.cpp.o
[  2%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodeUpcBase.cpp.o
[  2%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodeUpcA.cpp.o
[  2%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodeEan13.cpp.o
[  2%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodePostnet.cpp.o
[  3%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodePostnet5.cpp.o
[  3%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodePostnet9.cpp.o
[  3%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodePostnet11.cpp.o
[  4%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodeCepnet.cpp.o
[  4%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodeOnecode.cpp.o
[  4%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodeDataMatrix.cpp.o
[  4%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/BarcodeQrcode.cpp.o
[  5%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/DrawingPrimitives.cpp.o
[  5%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/Renderer.cpp.o
[  5%] Building CXX object glbarcode/CMakeFiles/glbarcode.dir/QtRenderer.cpp.o
[  5%] Linking CXX static library libglbarcode.a
[  5%] Built target glbarcode
[  5%] Generating moc_Backends.cpp
Scanning dependencies of target Barcode
[  5%] Building CXX object backends/barcode/CMakeFiles/Barcode.dir/Backends.cpp.o
[  5%] Building CXX object backends/barcode/CMakeFiles/Barcode.dir/Style.cpp.o
[  6%] Building CXX object backends/barcode/CMakeFiles/Barcode.dir/GnuBarcode.cpp.o
[  6%] Building CXX object backends/barcode/CMakeFiles/Barcode.dir/QrEncode.cpp.o
[  6%] Building CXX object backends/barcode/CMakeFiles/Barcode.dir/Zint.cpp.o
[  6%] Building CXX object backends/barcode/CMakeFiles/Barcode.dir/moc_Backends.cpp.o
[  7%] Linking CXX static library libBarcode.a
[  7%] Built target Barcode
[  7%] Generating moc_Merge.cpp
Scanning dependencies of target Merge
[  7%] Building CXX object backends/merge/CMakeFiles/Merge.dir/Factory.cpp.o
[  7%] Building CXX object backends/merge/CMakeFiles/Merge.dir/Record.cpp.o
[  8%] Building CXX object backends/merge/CMakeFiles/Merge.dir/Merge.cpp.o
[  8%] Building CXX object backends/merge/CMakeFiles/Merge.dir/None.cpp.o
[  8%] Building CXX object backends/merge/CMakeFiles/Merge.dir/Text.cpp.o
[  9%] Building CXX object backends/merge/CMakeFiles/Merge.dir/TextCsv.cpp.o
[  9%] Building CXX object backends/merge/CMakeFiles/Merge.dir/TextCsvKeys.cpp.o
[  9%] Building CXX object backends/merge/CMakeFiles/Merge.dir/TextTsv.cpp.o
[  9%] Building CXX object backends/merge/CMakeFiles/Merge.dir/TextTsvKeys.cpp.o
[ 10%] Building CXX object backends/merge/CMakeFiles/Merge.dir/TextColon.cpp.o
[ 10%] Building CXX object backends/merge/CMakeFiles/Merge.dir/TextColonKeys.cpp.o
[ 10%] Building CXX object backends/merge/CMakeFiles/Merge.dir/TextSemicolon.cpp.o
[ 10%] Building CXX object backends/merge/CMakeFiles/Merge.dir/TextSemicolonKeys.cpp.o
[ 11%] Building CXX object backends/merge/CMakeFiles/Merge.dir/moc_Merge.cpp.o
[ 11%] Linking CXX static library libMerge.a
[ 11%] Built target Merge
[ 11%] Generating moc_Variables.cpp
[ 11%] Generating moc_Model.cpp
[ 11%] Generating moc_ModelObject.cpp
[ 12%] Generating moc_ModelBarcodeObject.cpp
[ 12%] Generating moc_ModelBoxObject.cpp
[ 12%] Generating moc_ModelEllipseObject.cpp
[ 13%] Generating moc_ModelImageObject.cpp
[ 13%] Generating moc_ModelLineObject.cpp
[ 13%] Generating moc_ModelShapeObject.cpp
[ 13%] Generating moc_ModelTextObject.cpp
[ 14%] Generating moc_PageRenderer.cpp
[ 14%] Generating moc_Settings.cpp
Scanning dependencies of target Model
[ 14%] Building CXX object model/CMakeFiles/Model.dir/Category.cpp.o
[ 15%] Building CXX object model/CMakeFiles/Model.dir/ColorNode.cpp.o
[ 15%] Building CXX object model/CMakeFiles/Model.dir/DataCache.cpp.o
[ 15%] Building CXX object model/CMakeFiles/Model.dir/Db.cpp.o
[ 15%] Building CXX object model/CMakeFiles/Model.dir/Distance.cpp.o
[ 16%] Building CXX object model/CMakeFiles/Model.dir/FileUtil.cpp.o
[ 16%] Building CXX object model/CMakeFiles/Model.dir/Frame.cpp.o
[ 16%] Building CXX object model/CMakeFiles/Model.dir/FrameCd.cpp.o
[ 17%] Building CXX object model/CMakeFiles/Model.dir/FrameContinuous.cpp.o
[ 17%] Building CXX object model/CMakeFiles/Model.dir/FrameEllipse.cpp.o
[ 17%] Building CXX object model/CMakeFiles/Model.dir/FramePath.cpp.o
[ 17%] Building CXX object model/CMakeFiles/Model.dir/FrameRect.cpp.o
[ 18%] Building CXX object model/CMakeFiles/Model.dir/FrameRound.cpp.o
[ 18%] Building CXX object model/CMakeFiles/Model.dir/Handles.cpp.o
[ 18%] Building CXX object model/CMakeFiles/Model.dir/Layout.cpp.o
[ 18%] Building CXX object model/CMakeFiles/Model.dir/Markup.cpp.o
[ 19%] Building CXX object model/CMakeFiles/Model.dir/Model.cpp.o

~/glabels-qt/model/Model.cpp: In member function ‘const glabels::model::Frame* glabels::model::Model::frame() const’:
~/glabels-qt/model/Model.cpp:187:29: error: ‘const class QList<glabels::model::Frame*>’ has no member named ‘constFirst’
    return mTmplate.frames().constFirst();
                             ^
~/glabels-qt/model/Model.cpp: In member function ‘glabels::model::Distance glabels::model::Model::w() const’:
~/glabels-qt/model/Model.cpp:238:41: error: ‘const class QList<glabels::model::Frame*>’ has no member named ‘constFirst’
    if ( auto* frame = mTmplate.frames().constFirst() )
                                         ^
~/glabels-qt/model/Model.cpp: In member function ‘glabels::model::Distance glabels::model::Model::h() const’:
~/glabels-qt/model/Model.cpp:254:41: error: ‘const class QList<glabels::model::Frame*>’ has no member named ‘constFirst’
    if ( auto* frame = mTmplate.frames().constFirst() )
                                         ^
model/CMakeFiles/Model.dir/build.make:318: set di istruzioni per l'obiettivo "model/CMakeFiles/Model.dir/Model.cpp.o" non riuscito
make[2]: *** [model/CMakeFiles/Model.dir/Model.cpp.o] Errore 1
CMakeFiles/Makefile2:280: set di istruzioni per l'obiettivo "model/CMakeFiles/Model.dir/all" non riuscito
make[1]: *** [model/CMakeFiles/Model.dir/all] Errore 2
Makefile:162: set di istruzioni per l'obiettivo "all" non riuscito
make: *** [all] Errore 2

```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -107,11 +107,11 @@ if (MINGW)
    set (CMAKE_PREFIX_PATH ${MINGW_BASE_DIR} )
 endif ()
 
-find_package (Qt5Core 5.4 REQUIRED)
-find_package (Qt5Widgets 5.4 REQUIRED)
-find_package (Qt5PrintSupport 5.4 REQUIRED)
-find_package (Qt5Xml 5.4 REQUIRED)
-find_package (Qt5Svg 5.4 REQUIRED)
+find_package (Qt5Core 5.6 REQUIRED)
+find_package (Qt5Widgets 5.6 REQUIRED)
+find_package (Qt5PrintSupport 5.6 REQUIRED)
+find_package (Qt5Xml 5.6 REQUIRED)
+find_package (Qt5Svg 5.6 REQUIRED)
 find_package (Qt5LinguistTools)
 
 if (WIN32)
@@ -128,7 +128,7 @@ find_package (GnuBarcode 0.98 QUIET)
 find_package (LibQrencode 3.4 QUIET)
 find_package (LibZint 2.6 EXACT QUIET)
 # Unit testing support
-find_package (Qt5Test 5.4 QUIET)
+find_package (Qt5Test 5.6 QUIET)
 
 
 #=======================================

--- a/docs/BUILD-INSTRUCTIONS-LINUX.md
+++ b/docs/BUILD-INSTRUCTIONS-LINUX.md
@@ -6,7 +6,7 @@ gLabels Linux Build Instructions
 
 - g++
 - CMake 2.8.12+
-- Qt5 5.4+ Development Packages ( Qt5Core, Qt5Widgets, Qt5PrintSupport, Qt5Xml, Qt5Svg )
+- Qt5 5.6+ Development Packages ( Qt5Core, Qt5Widgets, Qt5PrintSupport, Qt5Xml, Qt5Svg )
 - zlib 1.2+ Development Package
 
 > Even if the above library packages are installed, their corresponding development packages

